corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -681,13 +681,13 @@ class ActionMixin(OrderedXmlObject):
 
 
 class Action(ActionMixin):
-    """ For CC < 2.21 """
+    """ For CC < 2.21.0 """
 
     display = NodeField('display', Display)
 
 
 class LocalizedAction(ActionMixin, TextOrDisplay):
-    """ For CC >= 2.21 """
+    """ For CC >= 2.21.0 """
```

Comment:
Specifying the `point` isn't necessary for any of these since we're only interested in when the feature was released (unless there's a specific case where we need to restrict it to after a bug was fixed)

Reply:
Ya, its just to bring the structure change. 2.21 should be 2.21.0

Reply:
This is minor, but I think 2.21 better corresponds to the mental model - conceptually this is a feature introduced in 2.21, not 2.21.0.  Technically yes, that was the first release in the minor version, but that's not the important part.  When talking about feature availability, it's only major and minor versions - think Django docs that say "Introduced in Django 1.10".

If you use `distutils.version.StrictVersion`, they're the same anyways:
```python
In [2]: StrictVersion('2.21') == StrictVersion('2.21.0')
Out[2]: True
```

Reply:
We might be stuck with `LooseVersion` to deal with the `".../Latest"` type business.  Either way, `LooseVersion('2.21') is always considered less than `LooseVersion('2.21.anything')`.  I think it makes sense to allow for omitting the the point release when checking a minimum version, but that anything declaring a specific app build should specify the point release.  I don't think it matters much one way or the other though, and I'm fine with this implementation.

Reply:
Cool, thanks @esoergel.

Reply:
I agree that it doesn't make a huge difference but my preference is also for the shorter version since they are semantically the same.